src/components/pages/TransactionHistoryPage.js

```diff
@@ -4,12 +4,48 @@ import Sidebar from "../reusable/SideBar";
 
 const TransactionHistoryPage = () => {
```

Comment:
This page is not as part of this story. This should not be included in this MR.

Reply:
I will take it out then.